src/app/navbar/navbar.component.css

```diff
@@ -93,5 +82,34 @@ i {
 
 .material-icons {
   margin: 0;
-  font-size: 70px;
+  font-size: 5rem;
```

Comment:
W tym przypadku defaultowa wartość analogicznie powinna wynosić mniejszy font-size niż wartość font-size w zakresie min-width: 320px. Dlatego że pierwszy breakpoint to 320px, a jeśli wyświetlimy aplikację na urządzeniu, które ma 319px to aktualnie font-size będzie wynosił 5rem.